README.md

```diff
@@ -1,6 +1,80 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio?style=social)](https://twitter.com/strimziio)
 
-# Strimzi Template repository
+# Awesome Strimzi
 
-This repository is currently empty
+A curated collection of **community-created** resources for [Strimzi](https://strimzi.io/)—Apache Kafka on Kubernetes. This repository hosts community contributions, including blog posts, tutorials, tools, SDKs, demos, and more.
+
+We use the popular **Awesome list** format to showcase valuable content created by contributors outside the official Strimzi organization. This gives visibility to community efforts while ensuring relevancy and quality.
+
+## Contents
+
+- [Official Resources](#official-resources)  
+- [Getting Started](#getting-started)  
+- [Articles & Blog Posts](#articles--blog-posts)  
+- [Talks & Presentations](#talks--presentations)  
+- [Libraries & SDKs](#libraries--sdks)  
+- [Tools & Utilities](#tools--utilities)  
```

Comment:
the links in the preview seem to work up to this one. Are the double hyphens needed?